teacher.rb

```diff
@@ -0,0 +1,26 @@
+# rubocop: disable Style/OptionalBooleanParameter
```

Comment:
- [ ] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `rubocop`, it will be nice to fix the issue.